apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -3,17 +3,27 @@
 /**
  * External dependencies
  */
+/* eslint-disable import/no-extraneous-dependencies */
 import domReady from '@wordpress/dom-ready';
 import ReactDOM from 'react-dom';
 import { __ } from '@wordpress/i18n';
 import { Button, Dashicon } from '@wordpress/components';
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * Internal dependencies
+ */
 import './style.scss';
 
 domReady( () => {
 	const { closeButtonLabel, closeButtonUrl, editorPostType } = fullSiteEditing;
 
 	// Only alter for the page and template part editors.
```

Comment:
not really important, but this comment could be updated now :D

Reply:
Definitely, done.